src/vs/workbench/contrib/terminalContrib/links/browser/terminalLinkParsing.ts

```diff
@@ -117,6 +117,8 @@ function generateLinkSuffixRegex(eolOnly: boolean) {
 		// foo: (339)
 		//   ...
 		`:? ?[\\[\\(]${r()}(?:, ?${c()})?[\\]\\)]` + eolSuffix,
+		// Support for '/path/to/file, 204'
+		`['"]?${p()}, ${r()}` + eolSuffix,
```

Comment:
Oh there's also a compile error here:

```
[tsec-compile-check      ] src/vs/workbench/contrib/terminalContrib/links/browser/terminalLinkParsing.ts(121,11): error TS2304: Cannot find name 'p'.
```

Note that r() is row, c() is column